### ISSUE TITLE
Event {isTrusted: true, type: "error", target: img, currentTarget: img, eventPhase: 2, …}

### ISSUE THREAD

**Issue by fuhui2017** [hilongjw/vue-lazyload#229]
![1](https://user-images.githubusercontent.com/28571205/36625140-bb76deb0-1954-11e8-9d32-48d9357a6c59.png)
how to deal with the error?
thank you !

**Comment by 143kk** [comment]
You are using v-lazy:background-image? Maybe you add url(....)

**Comment by McSneaky** [comment]
Had the same problem
Used image object like this: 
```js
image: {
  src: '', // This will be added later dynamically
  error: 'http://via.placeholder.com/350x150',
  loading: 'http://via.placeholder.com/350x150'
}
```
But when I changed image object to:
```js
image: {
  src: 'http://via.placeholder.com/350x150', // This will be changed later dynamically
  error: 'http://via.placeholder.com/350x150',
  loading: 'http://via.placeholder.com/350x150'
}
```
And all works as expected

**Comment by sanna-woo** [comment]
hi,

same problem here - since v.1.2.1

**Comment by McSneaky** [comment]
Have you tried adding image src at the beginning? It has to be actual URL. And then change it later on

**Comment by Wifsimster** [comment]
Why are you speak of `image` object ?

I simply use vue-lazy like that : `<img v-lazy="{url}">`

**Comment by McSneaky** [comment]
Can you set it url to some default image at the beginning and then change it later? In some method or mounted hook for example

**Comment by sanna-woo** [comment]
@McSneaky: 

Yes I did try this ...

I import vueLazyload as global component in my main.js file with following options set:

`import VueLazyload from 'vue-lazyload';
Vue.use(VueLazyload, {
    src: 'http://via.placeholder.com/350x250',
    error: require('../src/assets/img/image-not-found.jpg'),
    loading: require('../src/assets/img/ajax-spinner-ring.gif'),
})
`

in my components I then usually use it like this:

`<div class="preview lazy-img-fadein" v-lazy:background-image="getImageAsset(page)"></div>`

`getImageAsset(page) {
  return page.path + page.image ....
}`

**Comment by McSneaky** [comment]
Hmm, currently not behind computer to check out how I solved it. But something similar like that worked for me

**Comment by moshanghan** [comment]
升级到1.2.1就报错。还原到1.0.3版本，问题暂时解决，期待作者给出一个最优的解决方法

**Comment by sanna-woo** [comment]
any updates on this?

**Comment by hilongjw** [comment]
This should be an error log that failed to load the image. It was fixed in 1.2.2.

**Comment by vincent38wargnier** [comment]
Is it fixed? I still get the error on v 2.0.2 when i load large images

**Comment by HimanshuPrimo** [comment]
> ![1](https://user-images.githubusercontent.com/28571205/36625140-bb76deb0-1954-11e8-9d32-48d9357a6c59.png) how to deal with the error? thank you !

I am Facing this same Error How can I deal with this? Will anyone Available for Help
